test/inductor/test_triton_kernels.py

```diff
@@ -419,22 +429,35 @@ def call_triton(x: torch.Tensor, y: torch.Tensor, output: torch.Tensor):
         torch_add = call_triton(t1, t2, o1)
         metrics.reset()
         o2 = torch.zeros_like(t1, requires_grad=grad)
-        test, codes = run_and_get_code(
+        test, (code,) = run_and_get_code(
             torch.compile(call_triton, dynamic=dynamic), t1, t2, o2
         )
         if not grad:
             self.assertEqual(metrics.generated_kernel_count, 1)
         self.assertEqual(torch_add, test)
         # These two asserts are not optimal since it requires original aten
         # to be in the metadata, so there might be false negatives
-        self.assertTrue("aten.copy" not in codes[0])
-        self.assertTrue("aten.clone" not in codes[0])
+        self.assertNotIn(
+            "aoti_torch_copy_" if inductor_config.cpp_wrapper else "aten.copy", code
+        )
+        self.assertNotIn(
+            "aoti_torch_clone" if inductor_config.cpp_wrapper else "aten.clone", code
+        )
         # The following checks that there are only the tensor output is in
         # the compiled graph
         if dynamic and grad:
-            self.assertTrue("return (buf0, s0, )" in codes[0])
+            if inductor_config.cpp_wrapper:
+                self.assertIn("output_handles[0] = ", code)
+                self.assertIn("output_handles[1] = ", code)
```

Comment:
Should we also check buf0 and s0 here?

Reply:
@desertfire I don't think so? The two entries in `output_handles` represent `buf0` and `s0` here.